python/pylibcudf/pylibcudf/io/orc.pyi

```diff
@@ -39,3 +46,25 @@ class ParsedOrcStatistics:
 def read_parsed_orc_statistics(
     source_info: SourceInfo,
 ) -> ParsedOrcStatistics: ...
+
+class OrcWriterOptions:
+    def __init__(self): ...
+    def set_stripe_size_bytes(self, size_bytes: int) -> None: ...
+    def set_stripe_size_rows(self, size_rows: int) -> None: ...
+    def set_row_index_stride(self, stride: int) -> None: ...
+    @staticmethod
+    def builder(sink: SinkInfo, table: Table) -> OrcWriterOptionsBuilder: ...
+
+class OrcWriterOptionsBuilder:
+    def __init__(self): ...
+    def compression(
+        self, comp: CompressionType
+    ) -> OrcWriterOptionsBuilder: ...
+    def enable_statistics(
+        self, val: StatisticsFreq
+    ) -> OrcWriterOptionsBuilder: ...
+    def key_value_metadata(self, kvm: object) -> OrcWriterOptionsBuilder: ...
```

Comment:
Just as an aside, if you want to write a typed signature that has no particular restrictions on the argument/value, prefer to use `typing.Any` rather than `object`. It's OK here, because you're not providing a body to the function. But if you type with `object` to say "I'll accept anything", then you can (in the body) only access attributes of `object` (maybe that's what you want, but generally it isn't).

Similarly, using `Any` avoids the co-variance problem for function return types. If you want to say "the return value from this function has no particular restrictions", you can't say `object`, because then the only things you're allowed to do with the return value are exactly those things that are required by `object` (`object` is not a a subtype of anything).

So, if you want no restrictions on the inputs/outputs, write:

```
def foo(in_: Any) -> Any: ...
```